index.html

```diff
@@ -4521,8 +4521,9 @@
             <dt><dfn><code>fecMechanisms</code></dfn> of type <span class=
             "idlMemberType">sequence&lt;<a>DOMString</a>&gt;</span></dt>
             <dd>
-              <p>Supported Forward Error Correction (FEC) mechanisms. Values include
-              "red" [[!RFC2198]], "red+ulpfec" [[RFC5109]] and "flexfec" [[FLEXFEC]].
+              <p>Supported Forward Error Correction (FEC) mechanisms
+              and combinations. Values include "red" [[!RFC2198]],
+              "red+ulpfec" [[RFC5109]] and "flexfec" [[FLEXFEC]].
```

Comment:
May be worth mentioning that if these mechanisms are supported, corresponding codecs will appear in RtpCapabilities.codecs.